org.flathub.flatpak-external-data-checker.yml

```diff
@@ -0,0 +1,142 @@
+id: org.flathub.flatpak-external-data-checker
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+finish-args:
+  - --share=network
+  - --filesystem=host:ro
+command: flatpak-external-data-checker
+cleanup:
+  - "*.a"
+  - "*.la"
+  - /include
+  - /lib/pkgconfig
+  - /share/man
+  - /share/doc
+modules:
+
+  - name: flatpak-external-data-checker
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib /app/bin
+      - cp -ax . /app/lib/fedc
+      - ln -sr /app/{lib/fedc,bin}/flatpak-external-data-checker
+      - ln -sr /app/{lib/fedc,bin}/canonicalize-manifest
+      - install -Dm644 data/${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo/
+    sources:
+      - type: dir
+        path: "."
+    modules:
+
+      - name: python-apt
+        buildsystem: simple
+        build-commands:
+          - python3 setup.py build
+          - python3 setup.py install --prefix=/app --root=/
+        sources:
+          - type: archive
+            url: "http://deb.debian.org/debian/pool/main/p/python-apt/python-apt_1.8.4.tar.xz"
```

Comment:
Failed to download sources: module python-apt: The requested URL returned error: 404 Not Found

I appreciate the irony of this module itself going stale and needing to update itself.

Reply:
Indeed, why not allow the data-checker check itself.